components/button/button.js

```diff
@@ -7,9 +7,19 @@ const { PropTypes } = React;
 /**
  * General Button component. Use when you need button or a link that looks like button
  */
-function Button({ children, mods = [], size, href, onClick, type, variation, disabled, dataAttrs = {} }) {
+function Button(props) {
+  const {
+    children,
+    size,
+    href,
+    onClick,
+    type,
+    variation,
+    disabled,
+    dataAttrs = {},
+  } = props;
   const restProps = getDataAttributes(dataAttrs);
-
+  const mods = props.mods ? props.mods.slice() : [];
```

Comment:
I think it's a bad idea to have an arbitrary mods prop being passed here, specially considering there's no documentation for it and from what I see, should only affect internal behaviour (via the size and variation props)